src/ansys_sphinx_theme/extension/linkcode.py

```diff
@@ -0,0 +1,253 @@
+"""Module containing ``link_code`` extension."""
+import inspect
+import os
+import os.path as op
+import sys
+
+from docutils import nodes
+from docutils.nodes import Node
+import sphinx
+from sphinx import addnodes
+from sphinx.application import Sphinx
+from sphinx.locale import _
+
+DOMAIN_KEYS = {
+    "py": ["module", "fullname"],
+    "c": ["names"],
+    "cpp": ["names"],
+    "js": ["object", "fullname"],
+}
```

Comment:
This variable is repeated in the `__init__.py` file. If so, let us import it from there.

Reply:
In fact, I would suggest to add it at the `extension` sub-package level. The reason is to avoid any naming issues if importing the theme in the `conf.py`.